lib/ansible/cli/vault.py

```diff
@@ -384,7 +384,7 @@ def execute_encrypt_string(self):
                 sys.stderr.write(err)
             b_outs.append(to_bytes(out))
 
-        self.editor.write_data(b'\n'.join(b_outs), context.CLIARGS['output_file'] or '-')
+        self.editor.write_data(b'\n'.join(b_outs) + b'\n', context.CLIARGS['output_file'] or '-')
```

Comment:
I think a more elegant solution would be to add one more list element and `str.join()` will do the rest:
```suggestion
        b_outs.append(b'')
        self.editor.write_data(b'\n'.join(b_outs), context.CLIARGS['output_file'] or '-')
```

Reply:
@webknjaz Done